demo/app/lib/widgets/credential_metadata_card.dart

```diff
@@ -101,7 +109,36 @@ class CredentialMetaDataCard extends StatefulWidget {
                             color: Color(0xff6C6D7C),
                           ),
                           textAlign: TextAlign.start,
-                        )
+                        ),
+                      trailing: verifiedStatus ? const Text.rich(
+                        TextSpan(
+                          children: [
+                            WidgetSpan(child: Icon(Icons.verified_user_outlined,color: Colors.lightGreen, size: 18,)),
+                           TextSpan(
+                           text: 'Verified',
```

Comment:
The status check API says whether the credential is active or revoked. Can we change the text to active instead of verified?

Reply:
updated